analysis/civic/examples/transform/civic:aid6.json

```
@@ -0,0 +1,237 @@
+{
+    "statements": [
+        {
+            "id": "civic:aid6",
+            "description": "L858R is among the most common sensitizing EGFR mutations in NSCLC, and is assessed via DNA mutational analysis, including Sanger sequencing and next generation sequencing methods. Tyrosine kinase inhibitor afatinib is FDA approved, and is recommended (category 1) by NCCN guidelines along with erlotinib, gefitinib and osimertinib as first line systemic therapy in NSCLC with sensitizing EGFR mutation.",
+            "direction": "supports",
+            "evidence_level": "civic.amp_level:tier_i_-_level_a",
+            "proposition": "proposition:124",
+            "variation_descriptor": "civic:vid33",
+            "therapy_descriptor": "civic:tid146",
+            "disease_descriptor": "civic:did8",
+            "method": "method:002",
+            "support_evidence": [
+                "support_evidence:493",
+                "support_evidence:494",
+                "support_evidence:495",
+                "support_evidence:496",
+                "support_evidence:497",
+                "support_evidence:498",
+                "support_evidence:499"
+            ],
+            "type": "Statement"
+        }
+    ],
+    "propositions": [
+        {
+            "_id": "proposition:124",
```

```suggestion
            "id": "proposition:124",
```
one more change: let's get away from the VRS `_id` usage. We have a model that can support `value_id` separate from `value`, so we can do away with `_id` entirely; no need to mirror in propositions.